fictadvisor-api/src/v2/api/pipes/QuestionByRoleAndIdPipe.ts

```diff
@@ -9,13 +9,13 @@ export class QuestionByRoleAndIdPipe implements PipeTransform {
     private questionRepository: QuestionRepository,
   ) {}
 
-  async transform (params: { questionId: string, disciplineType: DisciplineTypeEnum }) {
-    const { questionId, disciplineType } = params;
+  async transform (params: { questionId: string, questionRole: DisciplineTypeEnum }) {
+    const { questionId, questionRole } = params;
     const question = await this.questionRepository.findById(questionId);
     if (!question) {
       throw new InvalidEntityIdException('question');
```

Comment:
Entity name must start with capital letter
```suggestion
      throw new InvalidEntityIdException('Question');
```